pkg/kubelet/kubelet.go

```diff
@@ -111,15 +115,13 @@ import (
 	sysctlwhitelist "k8s.io/kubernetes/pkg/security/podsecuritypolicy/sysctl"
 	utildbus "k8s.io/kubernetes/pkg/util/dbus"
 	utilipt "k8s.io/kubernetes/pkg/util/iptables"
-	"k8s.io/kubernetes/pkg/util/mount"
+	mounttypes "k8s.io/kubernetes/pkg/util/mount/types"
 	nodeutil "k8s.io/kubernetes/pkg/util/node"
 	"k8s.io/kubernetes/pkg/util/oom"
 	"k8s.io/kubernetes/pkg/util/selinux"
 	"k8s.io/kubernetes/pkg/volume"
 	"k8s.io/kubernetes/pkg/volume/csi"
 	"k8s.io/kubernetes/pkg/volume/util/subpath"
-	utilexec "k8s.io/utils/exec"
```

Comment:
Curious how this change relates to your pr?

Reply:
You are right that it doesn't. I tried to describe why when I opened the PR:

> Since I touched a lot of files outside of mount to use the new imports, and I was editing imports, I also followed the goimports standard of putting imports in order of: (1) go std libs (2) 3rd party (3) k8s.io 3rd party (4) k8s.io/kubernetes. The lack of consistency around this was getting to me.

I understand, though, if people want me to change it back. Part of it for me is that I have been coming at this from a mindset of moving a package **out** of `k/k`, and with that mind, it's been really useful to glance at a file and see if there are any `k/k` imports... When the imports aren't correctly grouped, it's hard to tell that. But that's just me being pedantic.